<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperatorWindow</class>
 <widget class="QMainWindow" name="OperatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="cred_oper_tab">
       <attribute name="title">
        <string>Персональные данные</string>
       </attribute>
       <widget class="QLabel" name="name_oper_label">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>140</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="email_oper_label">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>180</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLabel" name="name_oper_value">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>140</y>
          <width>311</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="email_oper_value">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>180</y>
          <width>291</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="phone_oper_label">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>210</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Номер телефона</string>
        </property>
       </widget>
       <widget class="QLabel" name="phone_oper_value">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>210</y>
          <width>291</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="edit_edit_oper_button">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>360</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Редактировать данные</string>
        </property>
       </widget>
       <widget class="QLabel" name="change_password_oper_label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>320</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Введите новый пароль</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="change_password_oper_value">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>320</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="save_password_oper_button">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>320</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="change_password_oper_button">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>310</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Изменить пароль</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="books_oper_tab">
       <attribute name="title">
        <string>Книги</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="edit_info_button">
             <property name="text">
              <string>Изменить информацию о книге</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="add_book_button">
             <property name="text">
              <string>Добавить книгу</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="clients_cb"/>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>Нет в наличии</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="percent_value">
             <property name="minimum">
              <double>0.010000000000000</double>
             </property>
             <property name="maximum">
              <double>9.990000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="change_perc_button">
             <property name="text">
              <string>Изменить на указ. %</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="change_squad">
             <property name="text">
              <string>Уменьшить на 25%</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="edit_book_titles_cb"/>
           </item>
           <item>
            <widget class="QSpinBox" name="edit_amount_sb">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>300</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="edit_price">
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
             <property name="maximum">
              <double>100000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="edit_book_save_button">
             <property name="text">
              <string>Сохранить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTableWidget" name="books_table_widget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Автор</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Жанр</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Количество</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Год публикации</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Цена</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Аннотация</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="authors_oper_tab">
       <attribute name="title">
        <string>Авторы</string>
       </attribute>
       <widget class="QTableWidget" name="authors_tabel">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>861</width>
          <height>471</height>
         </rect>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Имя</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата рождения</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата смерти</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="add_author_button">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>510</y>
          <width>100</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить автора</string>
        </property>
       </widget>
       <widget class="QLabel" name="name_new_label">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>510</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="birthdate_new_label">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>560</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Дата рождения</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="deathdate_new_value">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>580</y>
          <width>110</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="deathdate_new_label">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>560</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Дата смерти (опционально)</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="birthdate_new_value">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>580</y>
          <width>110</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="is_new_alive_checkbox">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>580</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Ныне живущий</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="name_new_value">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>530</y>
          <width>191</width>
          <height>24</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>100</number>
        </property>
       </widget>
       <widget class="QPushButton" name="save_new_author">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>530</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="genres_oper_tab">
       <attribute name="title">
        <string>Жанры</string>
       </attribute>
       <widget class="QTableWidget" name="genre_table">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>101</width>
          <height>231</height>
         </rect>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Название</string>
         </property>
        </column>
       </widget>
       <widget class="QLineEdit" name="genre_new_value">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>350</y>
          <width>191</width>
          <height>24</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>100</number>
        </property>
       </widget>
       <widget class="QLabel" name="genre_new_label">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>330</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
       <widget class="QPushButton" name="save_new_genre">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>350</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_genre_button_">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>330</y>
          <width>100</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить жанр</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cancel_new_genre">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>350</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="edit_genre_button">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>420</y>
          <width>121</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Изменить название</string>
        </property>
       </widget>
       <widget class="QComboBox" name="edit_genre_cb">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>420</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_genre_value">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>420</y>
          <width>113</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="edit_genre_cb_label">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>400</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Изменяемый жанр</string>
        </property>
       </widget>
       <widget class="QLabel" name="edit_genre_label">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>400</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Новое название</string>
        </property>
       </widget>
       <widget class="QPushButton" name="edit_genre_save_button">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>420</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="orders_oper_tab">
       <attribute name="title">
        <string>Заказы</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableWidget" name="all_orders">
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Номер заказа</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Номер телефона</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Email</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Стоимость</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата оформления</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Статус</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="statuses_cb"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
